tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -107,10 +112,26 @@ def test_positive_inventory_recommendation_sync():
 
     :BZ: 1957186
 
-    :CaseAutomation: NotAutomated
+    :CaseAutomation: Automated
 
     :CaseLevel: System
     """
+    org, ak = organization_ak_setup
+    cmd = f'organization_id={org.id} foreman-rake rh_cloud_insights:sync'
+    timestamp = datetime.utcnow().strftime('%Y-%m-%d %H:%M')
+    result = rhcloud_sat_host.execute(cmd)
+    wait_for(
```

Comment:
You could consider using `wait_for_tasks` here, although I think that might require you to get the task id rather than searching for `Insights full sync...`. So this implementation might be cleaner as is.

Reply:
Yes, that could be the case, but instead, I chose the timestamp to segregate the required one just to be more clear the Insights full sync gets done ...and the hosts hit data.

Reply:
Awesome just wanted to make sure you were aware of the `wait_for_tasks` function